src/train.py

```diff
@@ -0,0 +1,30 @@
+"""Train the model and export it to later use in prediction."""
+import pandas as pd
+from joblib import dump, load
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.model_selection import train_test_split
+from sklearn.naive_bayes import GaussianNB
+
+# load the preprocessed data
+corpus = load("output/preprocessed_data.joblib")
+dataset = pd.read_csv("output/dataset.csv")
+
+# Data transformation
+cv = CountVectorizer(max_features=1420)
```

Comment:
I guess this has been copied from Luis' project, right?

Reply:
Yes the model itself and its training was just taken from their project